### ISSUE TITLE
DMs: Color difference between avatar in room list and avatar in timeline

### ISSUE THREAD

**Issue by steef435** [vector-im/riotX-android#1422]
When you have a DM with an account without an avatar, the colour of the avatar in the room list may differ from the colour in the timeline.

![image](https://user-images.githubusercontent.com/1906214/83209416-98bdf000-a158-11ea-84d0-312c3a1e11eb.png)
![image](https://user-images.githubusercontent.com/1906214/83209426-9d82a400-a158-11ea-8824-062e6991a7f8.png)

Since an mxid has the same colour in all rooms it helps the memory if the avatar in the room list has the same colour as well. Not a big issue but thought I'd mention it anyway.

**Comment by bmarty** [comment]
Yes, it could be improve. Actually we use 3 colors for Avatars and 8 for usernames.

@nadonomy could you propose something here? Also any change will have to be implemented on all Riot clients

**Comment by steef435** [comment]
Web currently does use the same color for room and avatar when it comes to DMs it seems, though it's not the same as the username color:
![image](https://user-images.githubusercontent.com/1906214/83262377-2af7df80-a1bd-11ea-9e73-5ed00775bcb7.png)

**Comment by notramo** [comment]
Manually setting a color for the user (similarly to FB Messenger) could be a solution.
